content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,327 @@
+fields that can be configured.
+
+.. image:: create_quotations/other-info-delivery.png
+   :align: center
+   :alt: The Delivery section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Shipping Weight`: Displays the weight of the items being shipped. This field is not
+  modifiable. Product weight is configured on individual product forms.
+- :guilabel:`Incoterm`: Select an Incoterm (International Commerical Term) to use as predefined
+  commerical terms for international transactions.
+- :guilabel:`Incoterm Location`: If an Incoterm is being used, enter the international location in
+  this field.
+- :guilabel:`Shipping Policy`: Select a desired shipping policy from the drop-down menu. If all
+  products are delivered at once, the delivery order is scheduled, based on the greatest product
+  lead time. Otherwise, it is based on the shortest lead time. The available options are:
+  :guilabel:`As soon as possible` or :guilabel:`When all products are ready`.
+- :guilabel:`Delivery Date`: Click into the empty field to reveal a calendar popover, from which a
+  customer delivery date can be selected. If no custom date is required, refer to the
+  :guilabel:`Expected` date listed to the right of that field.
+
+Invoicing section
+~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Invoicing` section of the :guilabel:`Other Info` tab, there are invoicing specific
+fields that can be configured.
+
+.. image:: create_quotations/other-info-invoicing.png
+   :align: center
+   :alt: The Invoicing section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Fiscal Position`: Select a fiscal position to be used to adapt taxes and accounts for
+  particular customers or sales orders/invoices. The default value comes from the customer. If a
+  selection is made in this field, an :icon:`fa-refresh` :guilabel:`Update Taxes` clickable link and
+  icon appear. When clicked, the taxes for this partiuclar customer and quotation are updated. A
+  confirmation window appears, as well.
+- :guilabel:`Analytic Account`: Select an analytic account to apply to this customer/quotation.
+
+Tracking section
+~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Tracking` section of the :guilabel:`Other Info` tab, there are tracking specific
+fields that can be configured.
+
+.. image:: create_quotations/other-info-tracking.png
+   :align: center
+   :alt: The Tracking section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Source Document`: Enter the reference of the document that generated the
+  quotation/sales order, if applicable.
+- :guilabel:`Opportunity`: Select the specific opportunity (from the **CRM** app) related to this
+  quotation, if applicable.
+- :guilabel:`Campaign`: Select the marketing campaign related to this quotation, if applicable.
+- :guilabel:`Medium`: Select the method by which this quotation originated (e.g. *Email*,
+  *Facebook*, etc.), if applicable.
```

Comment:
Specific platforms like *Facebook* are actually **Source** values. *Email* is an acceptable **Medium** though. 

[Here's a doc](https://www.linkedin.com/pulse/whats-difference-between-source-medium-utm-parameters-/) that does a good job covering the difference between the two.

**Source** examples: facebook, google, bing, quora, youtube, reddit, etc.
**Medium** examples: cpc, organic-social, paid-social, email, email, etc.

Reply:
This section might be a good opp to do a **seealso** or a **tip** to use the link tracker app which has a form to generate UTM tracking values.